library/BIGBITMAPS

```diff
@@ -84,6 +88,10 @@
        '6)
 (DEFINEQ
 
+(BIGBITMAPP
```

Comment:
I suppose these changes are to avoid the cross-file record access. Sorry that just became clear.